index.html

```diff
@@ -29,26 +29,33 @@
         </div>
         <div class="hero">
             <nav>
-                <h2 class="logo"><span>R</span>.compaign</h2>
-                <ul>
-                    <li><a href="#">About</a></li>
+                <h2 class="logo"><a href=""><span>R</span>.company</a></h2>
+                <ul class=".nav-menu1">
+                    <li><a href="about.html">About</a></li>
                     <li><a href="#">Join</a></li>
                     <li><a href="#">Program </a></li>
                     <li><a href="#">Sponsor</a></li>
                     <li><a href="#">News</a></li>
-                    <button type="button">R.compaign</button>
+                    <button type="button">R.company</button>
                 </ul>
+                <button class="nav-icon" id="hamburger-btn">
+                    <i class="fas fa-bars"></i>
+                  </button>
+
+                  <button class="nav-icon" id="close-btn" >
+                    <i class="fas fa-times"></i>
+                  </button>
                 
             </nav>
            
         </div>
         <header >
             <div class="content">
-                <h3>"Hello!"</h3>
-                <h1>TechConnect <br> Web Development Workshop</h1>
-                <p class="frist-p">The TechConnect Web Development Workshop website aims to provide a platform for aspiring web developers to learn and enhance their skills in the field of web development. The website will offer a range of resources, tutorials, and interactive activities to help participants grasp the fundamental concepts of web development and stay updated with the latest industry trends.</p>
-                <p class="second-P">please contact us per Emeil for any further questions about TechConnect </p>
-                <a href="mailto:aminiroland01@gmail.com"><i class="email">aminiroland01@gmail.com</i></a>
+                <h3>"Hello!Let code"</h3>
+                <h1>TechConnect <br> Web Development<br> Workshop</h1>
+                <p class="frist-p">The TechConnect Web Development Workshop website aims to provide a platform for aspiring web developers to learn and enhance their skills in the field of web </p>
+                <h2 class="nation">2023.8.10(THU) ~ 16 (FRI)</h2>
+                <p class="nation">@ National Museum of Korea,Art Center Nabi and more</p>
     
             </div>
```

Comment:
- [ ] `Required`
- [ ] Good try 👍 
- [ ] The pages should look almost identical to the original design, outside the content differences.
- [ ] The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design
- [ ] Please make amends 🥂 
- [ ] see screenshots below

`Original Template`
![image](https://github.com/RolandAmini/Capstone-project/assets/66230306/08b89912-688e-4e03-bb94-8e46f1d4e07d)

`Your template`
![image](https://github.com/RolandAmini/Capstone-project/assets/66230306/3929f846-ec1b-4836-8e40-81821f212f5d)